sig_tracker/about_sig/urls.py

```diff
@@ -4,7 +4,7 @@
 app_name = 'about_sig'
 urlpatterns = [
     #/about_sig/
-    url(r'', views.index, name='index'),
+    url(r'^', views.index, name='index'),
```

Comment:
Did you try this out? Putting a caret here means the URL should start from this point. Therefore, the URL `/about_sig/` will not match any pattern.

Reply:
Wierd; It's working for me too. Never mind, thanks for checking. :)

Reply:
Here and below as well.